src/EventGridEmulator/EventGridEmulator.csproj

```diff
@@ -2,12 +2,17 @@
   <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
     <DebugType>none</DebugType>
   </PropertyGroup>
+  <PropertyGroup>
+    <UserSecretsId>03056322-03d7-4174-ac77-5a62892a7a87</UserSecretsId>
+    <DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
+  </PropertyGroup>
   
   <ItemGroup>
     <PackageReference Include="Azure.Messaging.EventGrid" Version="4.17.0" />
     <PackageReference Include="Microsoft.Extensions.Http" Version="7.0.0" />
     <PackageReference Include="Microsoft.Extensions.Http.Polly" Version="7.0.9" />
     <PackageReference Include="Microsoft.Extensions.Options" Version="7.0.1" />
+    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.19.5" />
```

Comment:
What is this one for and given it has VisualStudio in the name I guess it doesn't work with Rider?

Reply:
That was added by docker-compose project.